jpasskit/src/main/java/de/brendamour/jpasskit/signing/PKPassTemplateInMemory.java

```diff
@@ -66,55 +68,53 @@
     public static final String PK_PERSONALIZATION_LOGO_RETINA = "personalizationLogo@2x.png";
     public static final String PK_PERSONALIZATION_LOGO = "personalizationLogo.png";
 
-    private Map<String, InputStream> files = new ConcurrentHashMap<>();
+    private final Map<String, byte[]> files = new ConcurrentHashMap<>();
```

Comment:
As for me, I expected that you'd use `ByteBuffer` itself (the one which is used in `getAllFiles ` method's signature). Is there any benefit in using low-level array of `byte`?